### ISSUE TITLE
New to coding, how would I go about fetching balances for specific currencies? 

### ISSUE THREAD

**Issue by SuchUmami** [ccxt/ccxt#17368]
### ؜

Hi, I wrote a few pieces of code to first connect to my api and then to display my balance (I've blocked out the password and api keys). I imported the first module and when I ran it, it worked! Here is the code:

Here is the first module named dontshare_config001 https://imgur.com/a/wt9w1JD

Here is the second piece of code: https://imgur.com/wtLJCOb

This works well but it fetches all the currency balances. I'd like to simply fetch the balance for say bitcoin or USD. How would I go about doing this?

**Comment by carlosmiei** [comment]
Hello @SuchUmami,
First let's avoid posting images, they're hard to parse and read.

Second, `fetchBalance` returns all balances indexed by `currency` so you just need to access the balance use the currency as the key.


Example:
```Python
all_balances = await exchange.fetch_balance()
usdt_balance = all_balances['USDT"]
```

**Comment by SuchUmami** [comment]
> ```python
> all_balances = await exchange.fetch_balance()
> usdt_balance = all_balances['USDT"]
> ```

Thanks for your response! I am very new to coding (about a week in) so please forgive how noob some of my questions might be. Would I just copy/paste your example into the end of the second code?

Also, how did you create your example with all the code written there?
